btk/metrics.py

```diff
@@ -293,8 +294,8 @@ def compute_metrics(  # noqa: C901
         segmentations (list) : Contains the measured segmentations, as a list of boolean arrays of
                                shape MHW where M is the number of detected objects (must be
                                consistent with corresponding detection catalog)
-        deblended_images (list) : Contains the deblended images, as a list of arrays of shape MCHW
-                                or MHWC depending on dim_order, where M is the number of detected
+        deblended_images (list) : Contains the deblended images, as a list of arrays of shape NCHW
```

Comment:
I'm not sure that I agree with that change. I put M instead of N there to differentiate with the number of blends which we usually call N (and which would be the length of the list here).

Reply:
Yes, you are right, I hadn't read the description above. I just changed it back